tests/test_client.py

```diff
@@ -408,61 +393,43 @@ def create_modifier_list(self):
                                                                       'ordinal': 1}
                                                                  ],}
                                           }]}],
+
+    def create_item(self, num_records=1):
+        start_date = '2020-07-29T00:00:00Z'
```

Comment:
I think we said this, but I want to note it: we'll change this to use the start date in the test rather than be hard coded

Reply:
➕